### PR TITLE
Cronjob pod scheduleing updates

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- [update compute and highmem nodes](https://github.com/airqo-platform/AirQo-api/commit/a222f7184c34e481f44fab544b413fd301241ef1)
- [data-mgt-api node preferences](https://github.com/airqo-platform/AirQo-api/commit/74d5ea0b5359cbea05a1b6a9962d75f296cb4643)
- [schedule all hourly jobs only on compute node](https://github.com/airqo-platform/AirQo-api/commit/1f98ced4d001973b5b8e71bbf23c9b678eac9802)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- [PLATOPS-16](https://airqoteam.atlassian.net/browse/PLATOPS-16?atlOrigin=eyJpIjoiM2E5OWEzMWY0ZjA1NDVmMGI2Mzg3MjQ3MDQ3Y2VlMzkiLCJwIjoiaiJ9)

**_HOW DO I TEST OUT THIS PR?_**
- Study the code changes

**_ARE THERE ANY RELATED PRs?_**
- #1271 

[PLATOPS-16]: https://airqoteam.atlassian.net/browse/PLATOPS-16?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: k8s/nodes/prod-compute.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+  "kind": "Node",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "airqo-k8s-worker-0",
+    "labels": {
+      "node-type": "compute"
     }
-}
\ No newline at end of file
+  }
```
Comment (123MwanjeMike) [inline k8s/nodes/prod-compute.json]: [update compute and highmem nodes](https://github.com/airqo-platform/AirQo-api/pull/1279/commits/a222f7184c34e481f44fab544b413fd301241ef1)

**Review by 123MwanjeMike — COMMENTED** [review]
Guiding comments

File: k8s/data-mgt/templates/deployment.yaml [hunk 2]
```diff
@@ -35,7 +35,7 @@ spec:
                   - key: node-type
                     operator: In
                     values:
-                      - general-purpose
+                      - compute
```
[the remark this answers is not in the document]
Comment (Baalmart) [reply k8s/data-mgt/templates/deployment.yaml:38]: Hi @123MwanjeMike , please shed more light on this `compute` node?
Any quick notes about it?
Thanks in advance!

**Review by Baalmart — COMMENTED** [review]
Great job @123MwanjeMike , this is amazing!

Just left one review comment. :)

File: k8s/data-mgt/templates/deployment.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (123MwanjeMike) [reply k8s/data-mgt/templates/deployment.yaml:38]: Thanks @Baalmart 
The compute node is a node with much more computational resources than the rest. This means that it is better suited for scheduling on pods for deployments that are potentially resource intensive.

[the remark this answers is not in the document]
Comment (Baalmart) [reply k8s/data-mgt/templates/deployment.yaml:38]: Thanks @123MwanjeMike for the clarifications.

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @123MwanjeMike

**Comment by Baalmart** [comment]
Hi @123MwanjeMike , 
please see feedback in attached screenshot.

Screenshot 2023-02-01 at 12 13 44

**Comment by 123MwanjeMike** [comment]
> Hi @123MwanjeMike , please see feedback in attached screenshot.
> 
> Screenshot 2023-02-01 at 12 13 44

Hi @Baalmart 
The first one is a **preference** while the second one is a **requirement**

**Pull request merged**